### DISCUSSION TITLE
Why my docker builds show failing

### DISCUSSION THREAD

**Discussion by kilosonc** [badges/shields#6754] [Q&A] [answered]
https://hub.docker.com/r/casbin/casbin-server/builds

As you can see, all my builds results are success, but the badge shows failing.

![image](https://user-images.githubusercontent.com/52988625/125182174-5feab880-e23e-11eb-96fc-9c0b7843eb62.png)

**Comment by calebcartwright** [comment, marked as answer]
> Why my docker builds show failing

Because the Dockerhub API says the build is failing (details below for reference). This is something you'd need to take up with Docker.

We've seen issues with the Docker API in the past so I wouldn't be surprised if you find similar existing issue reports in the Docke repos/issue tracker

https://hub.docker.com/api/build/v1/source?image=casbin/casbin-server

```json
{
  "meta": {
    "limit": 25,
    "next": null,
    "offset": 0,
    "previous": null,
    "total_count": 1
  },
  "objects": [
    {
      "autotests": "OFF",
      "build_in_farm": true,
      "build_settings": [
        "/api/build/v1/setting/dc1115ff-fe98-4873-a490-afacbd234773/",
        "/api/build/v1/setting/27bc54c5-80de-4ab3-aa4e-d3972fd041ff/"
      ],
      "channel": "Stable",
      "image": "casbin/casbin-server",
      "owner": "casbin",
      "provider": "Github",
      "repo_links": false,
      "repository": "casbin-server",
      "resource_uri": "/api/build/v1/source/da89a1da-5f10-4d1c-9dc2-8efaef33c451/",
      "state": "Failed",
      "uuid": "da89a1da-5f10-4d1c-9dc2-8efaef33c451"
    }
  ]
}
```
